### ISSUE TITLE
`ResolverError` doesn't seem to be in use

### ISSUE THREAD

**Issue by fsat** [typesafehub/conductr-cli#422]
Is the `ResolverError` exception used? I can't seem to find where `ResolverError` is raised from.

The `ResolverError` is declared here:

https://github.com/typesafehub/conductr-cli/blob/master/conductr_cli/exceptions.py#L40

I guess if the `ResolverError` is meant to be raised by one of the resolver, the `load()` method in the `conduct_load.py` need to have validation handler for this (i.e. similar to `@validation.handle_bintray_resolution_error`)

**Comment by fsat** [comment]
Doing file search doesn't show the usage of the `ResolverError`

```
192-168-1-5:conductr-cli felixsatyaputra$ git show HEAD
commit 34533abe6f77ff48baac3f1e8170de9ce41be9da
Merge: b27aa31 0ab6f23
Author: Felix Satyaputra <felix_sat@yahoo.co.uk>
Date:   Wed Apr 19 12:53:18 2017 +1000

    Merge pull request #406 from longshorej/docker-resolver

    Docker resolver

192-168-1-5:conductr-cli felixsatyaputra$ find conductr_cli -iname "*.py" | xargs grep "ResolverError"
conductr_cli/exceptions.py:class ResolverError(Exception):
```

**Comment by longshorej** [comment]
Thanks for catching that - I see this is fixed via #426 now so closing.
